File: Java/Important Fundamentals/08. Leap year or not.java
import java.util.Scanner;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.println("Enter the Year:");
        int year = userInput.nextInt();

        if(( year % 400 == 0) || ((year % 4 == 0) && (year % 100 != 0))) {
            System.out.println(year + " is Leap Year");
        } else {
            System.out.println(year + " is not a Leap Year");
        }

        userInput.close();
    }
}

File: Python/01. Important Fundamentals/08. Leap year or not.py
year = int(input("Enter the year:"))

if(( year % 400 == 0 ) | (( year % 4 == 0 ) and ( year % 100 != 0 ))):
    print(year, "is Leap Year")
else:
    print(year, "is not a Leap year")

# Using Calendar Method
import calendar

year = int(input("Enter the year:"))

if calendar.isleap(year):
    print(year, "is Leap year")
else:
    print(year, "is not Leap year")